Project.toml

```diff
@@ -1,7 +1,11 @@
 name = "FixedSizeArrays"
 uuid = "3821ddf9-e5b5-40d5-8e25-6813ab96b5e2"
-authors = ["Mosè Giordano <mose@gnu.org>"]
```

Comment:
Personally I try to avoid unrelated changes like the lines being reordered here unnecessarily.

Reply:
I didn't change it, Pkg did it